setup.sh

```diff
@@ -1,14 +1,23 @@
 #!/usr/bin/env bash
 
-docker compose --ansi never up -d --build 2>& 1> /dev/null
-docker compose cp proxy:/usr/bin/caddy /tmp/caddy
+docker compose up -d 2>/dev/null
 
-XDG_DATA_HOME=/var/lib/docker/volumes/dev_caddy_data/_data
+docker compose exec proxy wget -q -O- --post-data='"0.0.0.0:2019"' --header='Content-Type:application/json' 'http://127.0.0.1:2019/config/admin/listen'
 
-sudo XDG_DATA_HOME=${XDG_DATA_HOME} \
-/tmp/caddy start 2&> /dev/null; \
-/tmp/caddy untrust; \
-/tmp/caddy trust; \
-/tmp/caddy stop 2&> /dev/null; \
+docker compose cp proxy:/usr/bin/caddy /tmp/caddy 2>/dev/null
 
-rm /tmp/caddy
+/tmp/caddy trust
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Why removing the `untrust` before trusting a new one. It's was here  to cleanup intermediary certificates ?